### ISSUE TITLE
New update?

### ISSUE THREAD

**Issue by lucian991** [UnamSanctam/UnamDownloader#55]
New update with download from pastebin/rentry?

**Comment by UnamSanctam** [comment]
Yes I have completed it but I haven't been able to make an undetected build from gcc, I've tried all different versions that I can find and even completely blank programs or normal "Hello World" programs get detected. So I'm trying a lot of different things right now.

**Comment by lucian991** [comment]
Can you send that current release, it doesnt matter if its not undetected

**Comment by lucian991** [comment]
I'd use it if its not a problem, until you post a public release

**Comment by lucian991** [comment]
So? @UnamSanctam

**Comment by lucian991** [comment]
Hello? @UnamSanctam

**Comment by UnamSanctam** [comment]
Well I'll have to put together it for release (all the resources and things like that) to be able to send it. First I'm focusing on the web panel, I'm planning to release the update today.

**Comment by lucian991** [comment]
Eh okay

**Comment by DogusCzk** [comment]
> Well I'll have to put together it for release (all the resources and things like that) to be able to send it. First I'm focusing on the web panel, I'm planning to release the update today.

today 😂😂
